.chloggen/mdatagen-filter.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: "metadatagen"
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "allow filtering attribute values using string and regexes"
```

Comment:
I suggest we'd better enhance this description, for example `support excluding some metrics based on string and regexes in resource_attributes`.